api/app.ts

```diff
@@ -29,6 +29,8 @@ app.post('/changePassword', changePasswordEndpoint);
 
 import accountCreate from './accounts/create';
 app.post('/accounts/create', accountCreate);
+import resetPassword from './accounts/resetPassword';
+app.post('/accounts/resetPassword', resetPassword);
```

Comment:
Should we convert these update functions to PUT instead of POST? (Since we are updating a value instead of creating a new one?)

Reply:
This cannot be PUT because it is not idempotent.